pom.xml

```diff
@@ -229,6 +229,10 @@
 			<artifactId>scenery</artifactId>
 			<version>0.1.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>net.imagej</groupId>
+			<artifactId>imagej-legacy</artifactId>
```

Comment:
Since you are only adding this to make a file dialog behave correctly for testing, please add it at test scope. Otherwise, you infect all downstream components with this spurious dependency. Alternately, if it _must_ be in runtime scope for some reason, you can put `<optional>true</optional>` to avoid this propagation. Either way, I highly doubt you want this at compile scope.